chsdi/templates/htmlpopup/cadastralwebmap_opendata.mako

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
```

Comment:
you could inherit the other cadastral webmap template or make this one inherit the other one. (like it's done with base.mako hereabove)